From abf86f8b564ff86ee54beeb9ac6332f881cc84f2 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Mon, 30 Nov 2015 13:22:05 -0600
Subject: [PATCH 3/4] arch/arm/mach-axxia: Update the Axxia GIC Driver

Replace the use of the depricated __this_cpu_ptr() with
raw_cpu_ptr().

git://git.yoctoproject.org/linux-yocto-4.1 standard/axxia/base
	commit f76bcc825c9ebbf4ebc3e8f3b993a0615c21f52c

Signed-off-by: John Jacques <john.jacques@intel.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/mach-axxia/axxia-gic.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-axxia/axxia-gic.c b/arch/arm/mach-axxia/axxia-gic.c
index bf57485..72c9657 100644
--- a/arch/arm/mach-axxia/axxia-gic.c
+++ b/arch/arm/mach-axxia/axxia-gic.c
@@ -741,11 +741,11 @@ static void gic_cpu_save(void)
 	if (!dist_base || !cpu_base)
 		return;
 
-	ptr = this_cpu_ptr(gic_data.saved_ppi_enable[this_cluster]);
+	ptr = raw_cpu_ptr(gic_data.saved_ppi_enable[this_cluster]);
 	for (i = 0; i < DIV_ROUND_UP(32, 32); i++)
 		ptr[i] = readl_relaxed(dist_base + GIC_DIST_ENABLE_SET + i * 4);
 
-	ptr = this_cpu_ptr(gic_data.saved_ppi_conf[this_cluster]);
+	ptr = raw_cpu_ptr(gic_data.saved_ppi_conf[this_cluster]);
 	for (i = 0; i < DIV_ROUND_UP(32, 16); i++)
 		ptr[i] = readl_relaxed(dist_base + GIC_DIST_CONFIG + i * 4);
 
@@ -767,11 +767,11 @@ static void gic_cpu_restore(void)
 	if (!dist_base || !cpu_base)
 		return;
 
-	ptr = this_cpu_ptr(gic_data.saved_ppi_enable[this_cluster]);
+	ptr = raw_cpu_ptr(gic_data.saved_ppi_enable[this_cluster]);
 	for (i = 0; i < DIV_ROUND_UP(32, 32); i++)
 		writel_relaxed(ptr[i], dist_base + GIC_DIST_ENABLE_SET + i * 4);
 
-	ptr = this_cpu_ptr(gic_data.saved_ppi_conf[this_cluster]);
+	ptr = raw_cpu_ptr(gic_data.saved_ppi_conf[this_cluster]);
 	for (i = 0; i < DIV_ROUND_UP(32, 16); i++)
 		writel_relaxed(ptr[i], dist_base + GIC_DIST_CONFIG + i * 4);
 
-- 
1.7.5.4

